lib/utils/__tests__/isStandardSyntaxCombinator.test.js

```diff
@@ -6,86 +6,60 @@ const selectorParser = require('postcss-selector-parser');
 
 describe('isStandardSyntaxCombinator', () => {
 	it('tag', () => {
-		rules('a {}', (func) => {
-			expect(isStandardSyntaxCombinator(func)).toBeTruthy();
-		});
+		expect(isStandardSyntaxCombinator(postcss.parse('a {}').first)).toBeFalsy();
```

Comment:
[note] `combinator('a {}')` returns `undefined`, that is, there is no combinator in this CSS code.
The previous expectation was not tested, so I've changed the expectation a bit.